pkgs/top-level/release-outpaths.nix

```diff
@@ -0,0 +1,77 @@
+# Vendored from:
+#   https://raw.githubusercontent.com/NixOS/ofborg/74f38efa7ef6f0e8e71ec3bfc675ae4fb57d7491/ofborg/src/outpaths.nix
+
+#!/usr/bin/env nix-shell
+# When using as a callable script, passing `--argstr path some/path` overrides $PWD.
+#!nix-shell -p nix -i "nix-env -qaP --no-name --out-path --arg checkMeta true --argstr path $PWD -f"
+{ checkMeta
+, path ? ./../..
+
+# used by pkgs/top-level/release-attrnames-superset.nix
+, attrNamesOnly ? false
```

Comment:
I guess this and the `lib/tests/release.nix` change is only meant to be here until ofborg switches to `release-attrpaths-superset.nix`? Because otherwise this file would be better inlined

Reply:
Yes!  Exactly.